civihr_default_theme/scss/pages/_admin_absence_tab.scss

```diff
@@ -0,0 +1,47 @@
+.chr_leave-absences-admin {
+  background: transparent;
+  box-shadow: none;
+  padding: 0;
+}
+
+.chr_absence-tab-page {
+
+  .button-container {
+    margin-bottom: 20px;
+  }
+
+  .tabs-container {
+
+    ul {
```

Comment:
ul is required to customize certain styles to match design like border bottom etc.,